### PR TITLE
Feature/hypervolume visualizations

### PR DESCRIPTION
**Pull request by aeivazi**
Add Multiobjective hypervolume-based visualization plots generations.

### CODE REVIEW

File: tests/visualization_test.py [hunk 1]
```diff
@@ -149,6 +150,32 @@ def test_multi_objective_visualization_two_objectives():
...
+    ]
+
+    evaluations_per_optimizer = {"test_optimizer": [evaluations]}
+
+    fig = hypervolume_over_iterations(
+        evaluations_per_optimizer,
+        objectives=(Objective("loss_1", False), Objective("loss_2", False)),
+        reference_point=[10, 0],
+    )
+    print(fig)
```
Comment (LGro) [inline tests/visualization_test.py:176]: Just out of curiosity, what is this print statement intended to do as part of the test?

**Review by LGro — APPROVED** [review]
Thanks a lot @aeivazi :relaxed:

**Review by sfalkner — APPROVED** [review]
Thanks @aeivazi for implementing this!

File: tests/visualization_test.py (hunk 1 above)
Comment (aeivazi) [reply tests/visualization_test.py:176]: ah, left over after debugging session... I will remove it ;)

**Pull request merged**